flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -266,7 +266,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
         do {
             let envelope = try parseJsonEnvelope(event)
-            SentrySDK.capture(envelope)
+            PrivateSentrySDKOnly.capture(envelope)
```

Comment:
This is currently the only use of a private API.
The deleted files were the workaround and the symlinked file.